docs/releases/1.16-NOTES.md

```diff
@@ -25,6 +25,10 @@ the notes prior to the release).
 
 * No required actions yet known.
 
+# Deprecations
+
+* The `kops/v1alpha1` API is deprecated and will be removed in kops 1.18.
```

Comment:
we need explanation also what this means in case of user. For normal user this does not tell much

Reply:
Are there any user considerations? The only thing I can think of is they wouldn't be able to use any v1alpha1 specs with `kops relpace -f`